vendor/assets/jquery_palette_color_picker/palette-color-picker.scss

```diff
@@ -1,145 +1,155 @@
+  background-size: cover;
+  /*background: url(palette-color-picker-icon.svg) center center no-repeat;*/
+  cursor: pointer;
+
+  box-shadow: 0 0 0 1px $cpfp-border;
+  /*outline: 1px solid $cpfp-border;*/
+  border: 2px solid #fff;
+}
+
+// Bubble
+.palette-color-picker-bubble {
+  display: none;
+  margin: 12px 0;
+  position: absolute;
+  bottom: 100%;
+  width: $cpfp-width;
+  padding: 10px 8px 8px 8px;
+  border: 1px solid $cpfp-border;
+  border-radius: 3px;
+  background-color: #fff;
+
+  // Arrow bottom
+  &:after,
+  &:before {
+  	top: 100%;
+  	left: 14px;
+  	border: solid transparent;
+  	content: ' ';
+  	height: 0;
+  	width: 0;
+  	position: absolute;
+  	pointer-events: none;
+  }
+  &:after {
+  	border-color: transparent;
+  	border-top-color: $cpfp-background;
+  	border-width: 8px;
+  	margin-left: -8px;
+  }
+  &:before {
+  	border-color: transparent;
+  	border-top-color: $cpfp-border;
+  	border-width: 9px;
+  	margin-left: -9px;
+  }
+
+  // Open below
+  &.downside {
+    bottom: auto;
+    top: 100%;
+
+    // Arow on top
+    &:after, &:before {
+      top: auto;
+    	bottom: 100%;
+    }
+    &:after {
+    	border-top-color: transparent;
+    	border-bottom-color: $cpfp-background;
+    }
+    &:before {
+      border-top-color: transparent;
+    	border-bottom-color: $cpfp-border;
+    }
+  }
+
+  .swatch {
+    position: relative;
+    display: inline-block;
+    font: 0/0 a;
+    width: 12.66%;
+    padding-top: 11.00%;
+    margin: 1% 2%;
+    outline: 1px solid $cpfp-border;
+    border: 2px solid #fff;
+    border-radius: 3px;
+    cursor: pointer;
+    overflow: hidden;
+
+
+    &.active {
+      outline-color: #000;
+
+      -webkit-box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.75);
+      -moz-box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.75);
+      box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.75);
+    }
+
+    &.clear {
+      background-color: #fff;
+      &:before {
+        position: absolute;
+        content: '';
+        display: block;
+        top: -1px;
+        left: 0;
+        width: 100px;
+        height: 2px;
+        background-color: #e00;
+
+        transform-origin: 0 0;
+        transform:  rotate(45deg);
+      }
+    }
+  }
+
+  // x2 width swatches
+  &.double {
+    .swatch {
+      width: 29.30%;
+      padding-top: 11.00%;
+      margin: 1% 2%;
+      &.clear {
+        &:before {
```

Comment:
Expected nesting depth to be no more than 3                              max-nesting-depth<br>Expected double colon pseudo-element notation                            selector-pseudo-element-colon-notation